design-toolkit/components/src/components/select/select.stories.tsx

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2025 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import Placeholder from '@accelint/icons/placeholder';
+import type { Meta, StoryObj } from '@storybook/react';
+import type { ReactNode } from 'react';
+import { Icon } from '../icon';
+import { Options } from '../options';
+import { Select } from './index';
+
+const meta: Meta<typeof Select> = {
+  title: 'Components/Select',
+  component: Select,
+  args: {
+    label: 'Label',
+    description: 'Helper text',
+    errorMessage: '',
+    size: 'medium',
+    isDisabled: false,
+    isInvalid: false,
+    isRequired: true,
+    autoFocus: true,
+    placeholder: 'Select...',
+    layoutOptions: {
+      estimatedRowHeight: 46,
+    },
```

Comment:
why is this needed?  when i remove it, things seem okay. when i make the row height super big, things get a little silly. e.g. this is set to 600

https://github.com/user-attachments/assets/4c27050b-714b-4347-8f9d-a5dd56d0ace7

it's a very contrived/ridiculous example, but just highlighting things can get funky